WegliKit/Sources/ImagesFeature/ImagesCore.swift

```diff
@@ -131,7 +143,40 @@ public let imagesReducer = Reducer<ImagesViewState, ImagesViewAction, ImagesView
     default:
       return .none
     }
-    
+
+  case .takePhotosButtonTapped:
```

Comment:
would be nice to have small test for the feature. If you need any assist let me know

Reply:
I hope to find some time soon to add tests.